components/UserSaysProperty.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react'
+
+const UserSaysProperty = ({ panel, onSaveUserSaysPropertiesClick, onUserSaysChange }) => (
```

Comment:
Exactly. I thought it once but decided to separate them coz they might be different in the future. There should be an action field on response right?